### ISSUE TITLE
Comments from initial review of `README.md`

### ISSUE THREAD

**Issue by jesseposner** [siv2r/bip-frost-signing#2]
>TODO will duplicating the neccessary notations here improve readability? rather than mentioning its link

Including the notations would be helpful because it makes clear which are being used, and is also useful to have in case the notation changes in the reference.

>1 | max_participants | integer | 2..n | maximum number of signers allowed in the signing process

We might want to note that this can't exceed the curve order.

>2 | min_participants | integer | 2..max_participants | minimum number of signers required to initiate the signing process

The precision of the RFC definition is nice:

```
   NUM_PARTICIPANTS is a positive non-zero integer which
   MUST be at least MIN_PARTICIPANTS but MUST NOT be larger than
   MAX_PARTICIPANTS, where MIN_PARTICIPANTS <= MAX_PARTICIPANTS,
   MIN_PARTICIPANTS is a positive non-zero integer and MAX_PARTICIPANTS
   MUST be a positive integer less than the group order.
   ``` 

>1 | id | integer | 1..max_participants | used to uniquely identify each participant, must be distinct from the id of every other participant
-- | -- | -- | -- | --
2 | secshareid | integer (scalar) | 1..n | signing key of a participant

Isn't the `id` a scalar value as well? From the RFC definition:

```
      An identifier, which is a NonZeroScalar value denoted i in the
      range [1, MAX_PARTICIPANTS] and MUST be distinct from the
      identifier of every other participant.
```

>3 | pubshareid | curve point (plain public key) | shouldn't be infinity point | public key associated with the above signing key

For the range, in addition to not being the the infinity point, we might also want to mention that the point must be in the subgroup of the generator.

**Comment by siv2r** [comment]
Trying to specify the correctness conditions for FROST keys using pseudo-math equations seemed unclear and imprecise. Therefore, I opted for an approach that specifies these conditions as pseudocode under the Algorithms section. The commit e514f2a26d152cf8544c7b499234e9fe8df0cb73 makes this change.

Yes, the RFC definitions are better, so I used them directly after making some modifications (refer to [key generation](https://github.com/siv2r/bip-frost-signing?tab=readme-ov-file#key-generation))

**Comment by jesseposner** [comment]
`Key generation for FROST signing is out of scope for this document. However, we specify the requirements that a key generation method must satisfy to be compatible with this signing protocol.`

We might want to link to ChillDKG as an example of a compatible DKG.

**Comment by siv2r** [comment]
Yes, I missed the "not" thanks. Good idea! I'll add a footnote for 2^32 (we basically assume `n` to be a 4 byte unsigned integer, rather than a scalar value).

**Comment by ismaelsadeeq** [comment]
> This property is called "identifiable aborts" and ensures that honest parties can assign blame to malicious signers who cause an abort in the signing protocol.

Malicious or incorrectly implemented signers.

**Comment by siv2r** [comment]
Closing this issue as all suggestions has been incorporated in the latest version of the BIP.
